mesh/src/vertex/Cylindrical2dVertexMesh.cpp

```diff
@@ -150,40 +152,100 @@ Cylindrical2dVertexMesh::Cylindrical2dVertexMesh(Cylindrical2dMesh& rMesh, bool
                     assert(dij>1e-5); //Sanity check
                     normal_vector /= dij;
 
-                    /*
-                     * Here all extra edges are split in two so always have unique triangulation.
-                     *
-                     *   ____  three nodes
-                     *  | /\ |
-                     *  |/__\|. edge of mesh
-                     *
-                     * Changing this causes issues with stitching the left and right back together.
+                    /*  
+                     * Here we only add one extra node.
+                     *  
+                     *  ____  one extra image node 
+                     *   /\  
+                     *  /__\ edge of mesh 
+                     * 
+                     * Changing this can cause issues with stitching the left and right back together.
                      */
-                    unsigned num_sections = 2;
-                    for (unsigned section=0; section<=num_sections; section++)
-                    {
-                        double ratio = ((double)section)/(double)num_sections;
-                        c_vector<double,2> new_node_location = normal_vector + p_node_a->rGetLocation() + ratio*edge;
-
-                        //Check if near other nodes (could be inefficient)
-                        bool node_clear = true;
-                        double node_clearance = 0.05;
-
-                        for (unsigned i=0; i<nodes.size(); i++)
-                        {
-                            double distance = norm_2(mpDelaunayMesh->GetVectorFromAtoB(nodes[i]->rGetLocation(), new_node_location));
-                            if (distance < node_clearance)
-                            {
-                                node_clear = false;
-                                //break;
-                            }
+                    
+                    double ratio = 0.5;
+                    c_vector<double,2> new_node_location = normal_vector + p_node_a->rGetLocation() + ratio*edge;
```

Comment:
formatting: space either side of *

Reply:
Done along with rest of the occurrences in the file.